sdks/python/apache_beam/io/parquetio.py

```diff
@@ -567,5 +567,6 @@ def _flush_buffer(self):
     size = 0
     for x in arrays:
       for b in x.buffers():
-        size = size + b.size
+        if b is not None:
+          size = size + b.size
```

Comment:
Thanks for tracking this down!

I wonder what changed in 0.17 to reveal this. Maybe the parquet writer (which we use to generate test data) wasn't eliding the null buffer before?

Reply:
Maybe the array factory function (pa.array()) is smarter (`x` there was created just several lines ago, using that factory function)